File: Modul 5/Bahasa Python (modul 5)/PRAK505-2310817320015-AlfisyahRinaFajriati.py
def Biodata(tahunLahir, Namaku, Asal):
  tahun_sekarang = 2023
  umur = tahun_sekarang - tahunLahir
  print(f"Perkenalkan nama saya: {Namaku}")
  print(f"Umur saya: {umur}")
  print(f"Saya adalah angkatan: {tahun_sekarang}")
  print(f"Asal saya dari: {Asal}")

if __name__ == "__main__":
 tahunLahir = int(input(""))
 Namaku = input("")
 Asal = input("")
 Biodata(tahunLahir, Namaku, Asal)

File: Modul 5/Bahasa C (modul 5)/PRAK505-2310817320015-AlfisyahRinaFajriati.c
#include <stdio.h>
void Biodata(int tahunLahir, char *Namaku, char *Asal){
int tahun_sekarang = 2023;
int umur = tahun_sekarang - tahunLahir;
printf("Perkenalkan nama saya: %s\n", Namaku);
printf("Umur saya: %d\n", umur);
printf("Saya adalah angkatan: %d\n", tahun_sekarang);
printf("Asal saya dari: %s\n", Asal);
}

int main() {
int tahunLahir;
 char Namaku[20], Asal[15];
 scanf(" %d",&tahunLahir);
 scanf(" %[^\n]%*c",&Namaku);
 scanf(" %[^\n]%*c",&Asal);
Biodata(tahunLahir, Namaku, Asal);
return 0;
}